src/sql/plot.py

```diff
@@ -371,10 +378,34 @@ def histogram(
                 f"Breaks given : {breaks}. When using breaks, please ensure that "
                 "breaks are strictly increasing."
             )
-        if bins:
+
+    if _are_numeric_values(binwidth):
+        if binwidth <= 0:
             raise exceptions.ValueError(
-                "Both bins and breaks are specified. Must specify only one of them."
+                f"Binwidth given : {binwidth}. When using binwidth, please ensure to "
+                "pass a positive value."
             )
+        binwidth = float(binwidth)
+    elif binwidth is None:
+        pass
+    else:
+        raise exceptions.ValueError(
+            f"Binwidth given : {binwidth}. When using binwidth, please ensure to "
+            "pass a numeric value."
+        )
+
+    specified_args = [
+        args
```

Comment:
```
specified_args = [
        args
        for args, specified in zip(
            ["bins", "breaks", "binwidth"], [bins, breaks, binwidth]
        )
        if specified
    ]
```

this block of code is a bit confusing. it would also generate combinations like (bin, bin), (breaks, breaks) etc.

Reply:
Also, this logic of checking whether multiple arguments are not specified can b a separate util function `validate_non_conflicting_arguments` which accepts a list of conflicting arguments

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/789/files#diff-653c15e6161c97f356be04e072838ed15014af275c06019a7692ab727fa44b97R503-R509): Added `validate_mutually_exclusive_args` in util.py

I'm not quite sure how it can generate (bin, bin)? It will add "bins" to the list if bins argument is not None. The same logic applies to breaks and binwidth arguments.